resources/META-INF/plugin.xml

```diff
@@ -13,7 +13,7 @@
     <idea-version since-build="201.6487.17"/>
```

Comment:
Will it be still compatible with 201.6487.17 (2020.1) ?

Reply:
If not, let's increase it to current public build you've built for

Reply:
I'm unsure about this.

Reply:
Can you please update to the current version then, please? So that users of old versions do not get broken updates